Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -31,11 +31,12 @@
     <MDocDocumentationDirectory>$(MSBuildThisFileDirectory)..\docs\en</MDocDocumentationDirectory>
     <DebugType>portable</DebugType>
     <Configurations>Debug;Release</Configurations>
+		<_IsRelease Condition=" '$(Configuration)'=='Release' And '$(OS)' == 'Windows_NT' ">true</_IsRelease>
```

Comment:
Can we describe a more specific condition for release builds? Maybe, does MSBuild on CI add any specific Properties?

Reply:
Maybe instead of the _IsRelease variable, we can use the `'$(TF_BUILD)' == 'True' Or '$(CI)' == 'True'`.

That only gets triggered on CI.

But, I was unable to reproduce your errors. Did you check out submodules with the repo? The docs repo is needed in order to generate the xml docs.

Reply:
Yep, Does this really work on mac?

Reply:
Any additional thoughts on this to push it through? @mattleibow ?